bin/setup

```diff
@@ -1,29 +1,40 @@
 #!/usr/bin/env ruby
-require 'pathname'
+require "pathname"
+require "fileutils"
+
+include FileUtils
+
+ENV["RAILS_ENV"]               ||= ENV["PASSENGER_APP_ENV"]
+ENV["RAILS_RELATIVE_URL_ROOT"] ||= ENV["PASSENGER_BASE_URI"]
+
+def system!(*args)
+  system(*args) || abort("\n== Command #{args} failed ==")
+end
 
 # path to your application root.
-APP_ROOT = Pathname.new File.expand_path('../../',  __FILE__)
+APP_ROOT = Pathname.new File.expand_path("../../",  __FILE__)
 
-Dir.chdir APP_ROOT do
+chdir APP_ROOT do
   # This script is a starting point to setup your application.
   # Add necessary setup steps to this file:
 
-  puts "== Installing dependencies =="
-  system "gem install bundler --conservative"
-  system "bundle check || bundle install"
+  puts "== Building Dashboard App =="
+  puts "RAILS_ENV               = #{ENV["RAILS_ENV"]}"
+  puts "RAILS_RELATIVE_URL_ROOT = #{ENV["RAILS_RELATIVE_URL_ROOT"]}"
 
-  # puts "\n== Copying sample files =="
-  # unless File.exist?("config/database.yml")
-  #   system "cp config/database.yml.sample config/database.yml"
-  # end
+  puts "\n== Installing dependencies =="
+  system("bin/bundle check &> /dev/null") || system!("bin/bundle install --deployment")
```

Comment:
From the site:

> The `--deployment` flag activates a number of deployment-friendly conventions:
>
> Isolate all gems into `vendor/bundle`
> Require an up-to-date `Gemfile.lock`
> If `bundle package` was run, do not fetch gems from rubygems.org. Instead, only use gems in the checked in `vendor/cache`

I think we want the first two. Since these are Rails apps we don't need or support `bundle package`.